kolibri/core/notifications/models.py

```diff
@@ -103,7 +103,7 @@ class LearnerProgressNotification(models.Model):
     notification_object = models.CharField(max_length=200, choices=NotificationObjectType.choices(), blank=True)
     notification_event = models.CharField(max_length=200, choices=NotificationEventType.choices(), blank=True)
     user_id = UUIDField()
-    classroom_id = UUIDField()
+    classroom_id = UUIDField()  # This can be either a Classroom or a LearnerGroup id
```

Comment:
this seems a bit misleading. maybe it would be more straightforward to simply add an optional `group_id` field rather than have `classroom_id` serve double-duty?

Reply:
instead of add, I would say rename it. We don't need another field. It's just the field name is ambiguous now.
@jonboiser do you think changing the name of the field at this stage of 0.12.x could be too dangerous or should we go ahead?